### PR TITLE
Main Website; WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds #12022

### PR DESCRIPTION
**Pull request by adilmp1**
## Description

This pull request fixes a small glitch where the panel color (#7dc5f2) didn't quite match the site accent color (#7dc6f2). To maintain consistency, I've adjusted the panel color to #7dc6f8, ensuring it aligns seamlessly with the overall site design.

### CODE REVIEW

**Comment by adilmp1** [comment]
> @adilmp1 please sign the CLA as mentioned in the comment

done ✅

**Comment by kfaseela** [comment]
@craigbox cc

File: src/sass/misc/_panel.scss [hunk 1]
```diff
@@ -119,11 +119,11 @@
         .panel-title { color: $accentColor; }
 
         .btn {
-            border-color: #7dc5f2;
-            color: #7dc5f2;
+            border-color: #7dc6f8;
```
Comment (craigbox) [inline src/sass/misc/_panel.scss:122]: It's not immediately obvious without reference to this issue why this is here, and why it was changed.

I might add a comment to say that this is basically `$whicheverVariable` but with contrast adjusted to meet WCAG thresholds.

Comment (adilmp1) [reply src/sass/misc/_panel.scss:123]: comment added

**Review by craigbox — APPROVED** [review]

File: src/sass/misc/_panel.scss (hunk 1 above)
Comment (craigbox) [reply src/sass/misc/_panel.scss:124]: The lint output shows you the expected formatting, which you'll have to fix.

(Sorry that it seems overzealous about a comment!)

Comment (kfaseela) [reply src/sass/misc/_panel.scss:124]: The PR is already merged, is there anything more to be fixed?

Comment (craigbox) [reply src/sass/misc/_panel.scss:124]: nope, i was only explaining the problem. (And I only saw the first commit...)

Normally we would cherry pick website fixes to the current branch but right now it's due to be released ~soon. @adilmp1 I would check in with @ericvn or on #release-1.21 on slack as to if this needs to be cherrypicked to the release-1.21 branch or if that hasn't happened yet.

(If you don't understand any of those terms, please ask there, or Faseela or myself.)

**Pull request merged**